stories/tree/__snapshots__/tree.stories.storyshot

```diff
@@ -1180,632 +1150,643 @@ exports[`Storyshots Components/Tree Expanded Tree 1`] = `
-                      >
-                        Edit
-                      </span>
-                      
-                                    
-                    </a>
-                    
-                                
-                  </li>
-                  
-                                
-                  <li
-                    class="fd-menu__item"
-                  >
-                    
-                                    
-                    <a
-                      class="fd-menu__link"
-                      href="#"
-                    >
-                      
-                                        
-                      <span
-                        class="fd-menu__title"
-                      >
-                        Lock
-                      </span>
-                      
-                                    
-                    </a>
-                    
-                                
-                  </li>
-                  
-                                
-                  <li
-                    class="fd-menu__item"
-                  >
-                    
-                                    
-                    <a
-                      class="fd-menu__link"
-                      href="#"
-                    >
-                      
-                                        
-                      <span
-                        class="fd-menu__title"
-                      >
-                        Duplicate
-                      </span>
-                      
-                                    
-                    </a>
-                    
-                                
-                  </li>
-                  
-                                
-                  <li
-                    class="fd-menu__item"
-                  >
-                    
-                                    
-                    <a
-                      class="fd-menu__link"
-                      href="#"
-                    >
-                      
-                                        
-                      <span
-                        class="fd-menu__title"
-                      >
-                        Delete
-                      </span>
-                      
-                                    
-                    </a>
-                    
-                                
-                  </li>
-                  
-                            
-                </ul>
-                
-                        
-              </nav>
-              
-                    
-            </div>
+          <span
+            class="fd-tree__text"
+          >
+            Level 1
+          </span>
+          
+        
+        </a>
+        
+        
+      </div>
+      
+    
+    </li>
+    
+
+  </ul>
+  
+
+
+  <br />
+  <br />
+  
+
+
+  <h3>
+    Tree with Expanded Level 6 and Active Tree Items (All Tree Items) 
```

Comment:
Rename suggestion: **Tree Level 6 (Active Items)** titles should be as simple as possible, and then clarified more in the description if needed.